src/Sarif.Viewer.VisualStudio.Core/SarifViewerPackage.cs

```diff
@@ -355,27 +355,25 @@ OleMenuCommand CreateDynamicMenuCommand(ErrorListMenuCommand command, int id)
                             return menuCommand;
                         }
 
-                        using (OleMenuCommandService mcs = this.GetService<IMenuCommandService, OleMenuCommandService>())
+                        OleMenuCommandService mcs = this.GetService<IMenuCommandService, OleMenuCommandService>();
```

Comment:
>OleMenuCommandService [](http://example.com/codeflow?start=24&length=21)

What happened to our old UI tests? Are they still running post-checkin?

Reply:
This involves VS shell plumbing, no way to test that I'm aware of.